oar/core/worksheet.py

```diff
@@ -616,6 +617,124 @@ def create_test_results_links(self):
             )
         )
 
+    # ========================================================================
+    # BLOCKING SECURITY ALERTS FUNCTIONALITY
+    # ========================================================================
+
+    def setup_blocking_sec_alerts_label(self):
+        """
+        Set up the blocking sec-alerts text label in cell A29
+        """
+        self._ws.update_acell(LABEL_BLOCKING_SEC_ALERTS, "Blocking sec-alerts")
+        logger.debug("Blocking sec-alerts label created in A29")
+
+    def update_blocking_sec_alerts_status(self, status):
+        """
+        Update the blocking sec-alerts dropdown status
+
+        Args:
+            status (str): Either "Yes" or "No"
+
+        Raises:
+            WorksheetException: If status is not "Yes" or "No"
+        """
+        if status not in ["Yes", "No"]:
+            raise WorksheetException("Status must be either 'Yes' or 'No'")
+
+        self._ws.update_acell(LABEL_BLOCKING_SEC_ALERTS_DROPDOWN, status)
+        logger.debug(f"Blocking sec-alerts status updated to: {status}")
+
+    def get_blocking_sec_alerts_status(self):
+        """
+        Get the current blocking sec-alerts status
+
+        Returns:
+            str: The current status ("Yes", "No", or None if not set)
+        """
+        try:
+            return self._ws.acell(LABEL_BLOCKING_SEC_ALERTS_DROPDOWN).value
+        except Exception as e:
+            logger.warning(f"Failed to get blocking sec-alerts status: {e}")
+            return None
+
+    def add_blocking_sec_alerts_to_others_section(self, has_blocking, blocking_advisories):
+        """
+        Add blocking security alerts status to Others column with hyperlinked advisory numbers
+
+        Args:
+            has_blocking (bool): Whether blocking alerts were found
+            blocking_advisories (list): List of advisories with blocking alerts
+
+        Returns:
+            bool: True if entry was added successfully
+        """
+        try:
+            if has_blocking:
+                # Create hyperlinked summary message for Others column
+                if len(blocking_advisories) == 1:
+                    # Single advisory - format as: "🔴 Blocking Sec-Alert: [12345]"
+                    advisory = blocking_advisories[0]
+                    advisory_id = str(advisory["errata_id"])
+                    hyperlink = self._to_hyperlink(util.get_advisory_link(advisory_id), advisory_id)
```

Comment:
this method cannot update multiple hyperlinks to single cell

Reply:
Hmm you are right. Probably I will have to use the customized method you have created.